Nef_3/include/CGAL/Nef_3/SNC_indexed_items.h

```diff
@@ -116,7 +116,8 @@ class SNC_indexed_items {
       init_ifacet = sl.init_ifacet;
       return *this;
     }
-
+    int new_index()
+    { index = Index_generator::get_unique_index(); return index; }
     void set_index(int idx = Index_generator::get_unique_index())
```

Comment:
Wouldn't it make sense to remove the default for the parameter `idx`?

Reply:
Yes. I just didn't want to break any existing code.

Reply:
If it is not documented, and not used then we can change it.